lib/server.js

```diff
@@ -0,0 +1,19 @@
+import { Server } from 'hapi'
+
+export default (port) => {
+  const server = new Server()
+
+  server.connection({port: port})
+
+  server.start(() => {
+    console.log(`Running at http://localhost:${port}`)
+  })
```

Comment:
It gets protocol and a few other things. Full output:

```{ created: 1451954262493,
  started: 1451954262513,
  host: 'grigori',
  port: 8000,
  protocol: 'http',
  id: 'grigori:2309:ij0o2v31',
  uri: 'http://grigori:8000',
  address: '0.0.0.0' }```

But http://grigori:8000 doesn't work, of course; I don't have a host by that name. Mysterious.